### ISSUE TITLE
Failed to load Logboard with SwiftPM

### ISSUE THREAD

**Issue by wuxiaohua1011** [shogo4405/HaishinKit.swift#800]
**Describe the bug**
Failed to load LogBoard using Swift Package Manager installation.

I am trying to build on iOS, specifically, iphone 12 pro. 
I am using the command listed [here](https://github.com/shogo4405/HaishinKit.swift#swift-package-manager) `https://github.com/shogo4405/HaishinKit.swift`

The error log after clicking the run button is as follows

```
dyld: Library not loaded: @rpath/Logboard.framework/Logboard
  Referenced from: /private/var/containers/Bundle/Application/FB583879-E1A9-4863-87E5-A94672CD3998/haishinkit.app/Frameworks/HaishinKit.framework/HaishinKit
  Reason: image not found
dyld: launch, loading dependent libraries
DYLD_LIBRARY_PATH=/usr/lib/system/introspection
DYLD_INSERT_LIBRARIES=/Developer/usr/lib/libBacktraceRecording.dylib:/Developer/usr/lib/libMainThreadChecker.dylib:/Developer/Library/PrivateFrameworks/GPUTools.framework/libglInterpose.dylib:/usr/lib/libMTLCapture.dylib:/Developer/Library/PrivateFrameworks/DTDDISupport.framework/libViewDebuggerSupport.dylib
```

**To Reproduce**
1. Go to Files ->  Swift Package -> Add Package Dependency
2. paste https://github.com/shogo4405/HaishinKit.swift in the search bar
3. leave all settings as default, should look like this: [https://imgur.com/mqJY2WC](https://imgur.com/mqJY2WC)
4. click the run button after import is done
5. see error

Note that I found that it runs on the Iphone 12 Pro simulator. However, when I plug in the physical Iphone 12 Pro and select the build target to the physical phone, it failes

**Expected behavior**
It should run without error

**Comment by wuxiaohua1011** [comment]
hmm, im able to get around it by doing the following:

```
2.1 Use Swift Package manager to download the HaishinKit package
    - in xcode, open `file -> swift packages -> add package dependencies`
    - enter url https://github.com/shogo4405/HaishinKit.swift
2.2 use Cartharge to install LogBoard (Yes, you will see it listed as sub dependencies of HaisinKit, but it doens't work unfortunately)
    - set up cartharge: https://github.com/Carthage/Carthage#adding-frameworks-to-an-application
    - execute `carthage update --platform iOS` in the project directory with controller.xcodeproj
    - it will fail, but its okay.
    - drag the `Logboard.framework` file from `Build/iOS` directory to under Framwork, Library, and Embeded content section of the `controller.xcodeproj` file
 ```

It seems to me that the issue is in the `Logboard` repository? Newbie to iOS dev, not quite sure what exactly is happening.

**Comment by shogo4405** [comment]
Thank you. I fixed it 48c78fb.
